### PR TITLE
Functionality For Carousel - Added React Portals

### PR DESCRIPTION
**Pull request by miicow**

### CODE REVIEW

File: client/src/components/photogrid/PhotoGrid.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  margin: '40px auto',
+};
+
+export default class PhotoGrid extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      images: [
+        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/aurora.jpg',
```
Comment (kkaiyang94) [inline client/src/components/photogrid/PhotoGrid.jsx:20]: Try passing down photos from top level down in order to ensure consistent data across all components.

File: client/src/components/PhotoCarousel.jsx [hunk 2]
```diff
@@ -15,18 +16,34 @@ export default class PhotoCarousel extends Component {
...
+        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/aurora.jpg',
+        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/city.jpg',
+        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/mountains.jpg',
+        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/redsky.jpg',
       ],
+      showModal: false,
     };
   }
 
+  handleClick = () => {
```
Comment (kkaiyang94) [inline client/src/components/PhotoCarousel.jsx:29]: We can consider using one function for changing the state of showModal.

const { showModal } = this.state;

const newStatus = !showModal;

this.setState({
  showModal: newStatus
})

**Review by kkaiyang94 — APPROVED** [review]
Looks good overall! If we can start to integrate our database, and pass that data down as props, it'll make sure we have consistent data throughout the app regardless of which restaurant we are currently on.

**Comment by tlindow** [comment]
@miicow Hey Michael, I just made some commits to your work branch I would like to merge these to the master. These changes help import css into bundle.js so that we can use the styles when the component is exported/imported to the proxy server.

Let me know if you have any questions. -Tyler

**Pull request merged**